docs/features/records.md

```diff
@@ -21,78 +21,144 @@ This new form is called a *caller-receiver default-argument*, and is allowed onl
 
 When an argument is omitted from an invocation of a function member for a corresponding optional parameter with a *caller-receiver default-argument*, the value of the receiver's member is implicitly passed. 
```

Comment:
It is already the case that you cannot have default values on ref parameters. This spec is not intended to change that.